test/regtest.py

```diff
@@ -105,10 +114,24 @@ def process_test(cmd, test, memory, verifier, expect, log_file):
   p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
   out, err  = p.communicate()
   elapsed = time.time() - t0
+  status = 0
+
+  bplfile = cmd[cmd.index('-bpl')+1]
+  with open(os.devnull, 'w') as devnull:
+    for f in checkbpl:
+      with open(bplfile) as bpl:
+        checker = subprocess.Popen(shlex.split(f), stdin=bpl, stdout=devnull, stderr=devnull)
```

Comment:
It took me some time to understand this :). Why are you invoking grep Linux command as a separate process instead of doing a string search directly from Python?